changelog/11785.trivial.rst

```diff
@@ -0,0 +1,7 @@
+Some were made to private functions which may affect plugins which access them:
+
+- ``FixtureManager._getautousenames()`` now takes a ``Node`` itself instead of the nodeid.
+- ``FixtureManager.getfixturedefs()`` now takes the ``Node`` itself instead of the nodeid.
+- The ``_pytest.nodes.iterparentnodeids()`` function is removed without replacement.
+  Prefer to traverse the node hierarchy itself instead.
```

Comment:
```suggestion
  Prefer to traverse the node hierarchy itself using ``_pytest.nodes.iterparentnodes()`` instead.
```

Reply:
Oh, I now realize that you might not want to advertise the internal method on purpose -- if that's the case feel free to ignore this.

Reply:
Right, I don't mean to expose it. I think it can be nice as a `Node` method, maybe later.